lib/matplotlib/path.py

```diff
@@ -280,8 +281,17 @@ def __deepcopy__(self, memo=None):
         Return a deepcopy of the `Path`.  The `Path` will not be
         readonly, even if the source `Path` is.
         """
-        # Deepcopying arrays (vertices, codes) strips the writeable=False flag.
-        p = copy.deepcopy(super(), memo)
+        memo = memo or {}
+        if id(self) in memo:
+            return memo[id(self)]
```

Comment:
AFAIK, `memo` is only needed for passing it to `deepcopy()`. So just remove any other use of `memo`.

Reply:
You're right that this check is not needed since `copy.deepcopy` does it